packaging/update-wiki.sh

```diff
@@ -27,19 +27,13 @@ chmod 0600 "$SSH_KEY"
 rm -rf "$HOME/openra-wiki"
 git clone git@github.com:OpenRA/OpenRA.wiki.git "$HOME/openra-wiki"
 
-mono --debug ../OpenRA.Utility.exe all --docs "$1" > "${HOME}/openra-wiki/Traits${TAG}.md"
-mono --debug ../OpenRA.Utility.exe all --weapon-docs "$1" > "${HOME}/openra-wiki/Weapons${TAG}.md"
-mono --debug ../OpenRA.Utility.exe all --lua-docs "$1" > "${HOME}/openra-wiki/Lua API${TAG}.md"
 mono --debug ../OpenRA.Utility.exe all --settings-docs "$1" > "${HOME}/openra-wiki/Settings${TAG}.md"
```

Comment:
Why not publish this on `docs.openra.net` too?

Reply:
This seems more end user centric.